dashboard/app/views/layouts/_header.html.haml

```diff
@@ -81,8 +75,8 @@
                 = link_to(image_tag('logo.png'), '/home', {id: "logo_home_link"})
               - else
                 = link_to(image_tag('logo.png'), CDO.code_org_url, {id: "logo_home_link"})
-            .project_info.hide_on_mobile
-          .header_middle.hide_on_mobile
+            .project_info
```

Comment:
What is the `project_info` things we are unhiding? Is it the create a project and remix buttons? Is it possible to share a screenshot of the before/after of what this looks like? Probably you could try on [studio.code.org/p/artist](https://studio.code.org/p/artist).

Reply:
No change at this stage to such levels because we continue hiding the header on levels when device width <= 500px.

Reply:
Circling back, it is a good question because there is a chance of changed behaviour.  If there is a place we are showing the project interface (attached programmatically to `project_info`) or progress (below `header_middle`) on levels that don't have a `level` or `script_level` then we might start showing them on mobile devices when we didn't before.  I don't think that's possible but would love @joshlory, @Hamms, or @balderdash to chime in if they can think of any such levels.